### ISSUE TITLE
Update for V8

### ISSUE THREAD

**Issue by marbman21** [damianharouff/whmcshook-recordlogin#2]
Can this be updated for WHMCS V8?

**Comment by damianharouff** [comment]
It likely can be, WHMCS v8 has hook functionality as far as I'm aware.

**Comment by marbman21** [comment]
Ok, let me rephrase, could you update this for V8?

**Comment by damianharouff** [comment]
No, but feel free to go ahead and update it yourself if desired. I would be willing to entertain a pull request for valid code, but otherwise your fork can then live on as the gospel.

**Comment by marbman21** [comment]
Thanks for nothin....
